draft-ietf-idr-link-bandwidth.xml

```diff
@@ -158,25 +162,54 @@
+        the transitivity type of the link bandwidth community, as well as Global
+        Administrator filed value and bandwidth value in (Local Administrator
+        filed), trough local policy.&nbsp;No more
         than one link bandwidth extended community SHALL be attached to a
         route.</t>
+        <t>An originator can attach link bandwidth community to BGP path in
+        egress processing to adj-RIB-out only or in ingress processing in which
+        case link bandwidth community is present in Local-RIB.</t>
+        <section anchor="Egress"
+               title="Originating in egress processing">
+          <t>When link bandwidth community is inserted in egress bgp path processing,
+          it is present in adj-RIB-out only and is send to BGP peers, regardless
+          of session type (interbal vs. external) and community transitivity bit.
+          The link bandwidth community has no impact on sender's forwarding
+          behavioiour.</t>
+        </section>
+        <section anchor="Ingress"
+                 title="Originating in ingress processing">
+          <t>When link bandwidth community is inserted in ingress bgp path processing,
+          it is present in Local-RIB, and:</t>
+          <ul spacing=normal>
+            <li>The transitive link bandwidth community is send to BGP peers, regardless
+            of session type (interbal vs. external)</li>
+            <li>The non-transitive link bandwidth community is send to internal BGP peers
+            only.</li>
+          </ul>
+          <t>Note: Implementation may provide configuration option (knob) to allow sending
+          of non-transitive link bandwidth communities to external BGP sessiosns.</t>
+          <t>Since link bandwidth community is present in LOCAL-RIB, it may be used by
+          sender to construct weighted ECMP forwarding behaviour (provided BGP
+          multipath is in use)</t>
+        </section>
       </section>
 
       <section anchor="Receiver"
                title="Receiver (Receiving link bandwidth community)&nbsp;">
         <t>A BGP receiver MUST be able to process link bandwidth community of
         both transitive or non-transitive type. The receiver SHOULD NOT flap
         or treat the route as malformed based on the transitivity of the link
-        bandwidth community.</t>
+        bandwidth community and/or BGP session type (internal vs. external).</t>
       </section>
 
       <section anchor="Re-advertisement" title="Re-advertisement Procedures">
         <section anchor="NextHopSelf"
                  title="Re-advertisement with Next hop Self">
           <t>When a BGP speaker re-advertises a route with the Link Bandwidth
           Extended Community and sets the next hop to itself, it SHOULD follow
-          the same procedures as outlined in <xref target="Originator"/>.</t>
+          the same procedures as outlined in <xref target="Ingress"/>.</t>
```

Comment:
I think this should remain "Originator" xref, since both ingress/egress cases may happen here also.